### ISSUE TITLE
Row highlight and keyboard navigation in cell editing mode

### ISSUE THREAD

**Issue by arrinmurr** [free-jqgrid/jqGrid#224]
Hi,

Looks like row highlight on click ~~and keyboard navigation with arrow keys~~ has stopped working in cell editing mode since version 4.13.0. These were still working in 4.12.1.

Here are some simple examples:
4.12.1 https://jsfiddle.net/0bsytotk/ (working)
4.13.0 https://jsfiddle.net/ub0y1z1t/
master https://jsfiddle.net/pzeL55tx/

Edit: Keyboard navigation actually works, it's just that the highlighting doesn't happen so it's hard to follow.

**Comment by OlegKi** [comment]
Thank you for the bug report. The problem exist after changing in the structure which describes the CSS classes of jQuery UI and Bootstrap which should be used. After the changes the most code was adjusted, but the 

I posted the corresponding fix (see [here](https://github.com/free-jqgrid/jqGrid/commit/1b8a4bcdfb52ab09468ec73d11b05c9f731929aa)) to GitHub. The demo https://jsfiddle.net/pzeL55tx/, which uses the code from GitHub is working now. Thus I close the issue. Please confirm that the problem is solved. If you would see still some problems, please reopen the issue.

**Comment by arrinmurr** [comment]
Looks good to me, thank you!
